tests/CopyPasteTests.cs

```diff
@@ -65,46 +71,68 @@ public void CopyPasteTableView()
         var copy = new CopyOperation(tvDesign);
         OperationManager.Instance.Do(copy);
 
-        ClassicAssert.AreEqual(0, OperationManager.Instance.UndoStackSize,
-            "Since you cannot Undo a Copy we expected undo stack to be empty");
+        // TODO: Remove this comment once addressed
+        // I think it would be a good idea to make the undoStack internal and then
+        // expose internals to the test project namespace, so we can actually inspect
+        // the collection itself.
+        // Using this proxy property only gives the test limited validity.
```

Comment:
I agree that this would allow more comprehensive testing.  I will create an issue.

Reply:
I've been doing it anyway, as I've been writing some tests, so you might not have to bother.

Just flipping them to internal and then adding the internalsvisibleto in the project file was all I had to do.